RabbitMQ.pm

```diff
@@ -81,26 +83,30 @@ will exit with error code 141
 
 =head2 disconnect()
 
-Causes the connection to RabbitMQ to be torn down.
+Disconnect from the RabbitMQ server.
 
 =head2 get_server_properties()
 
-Get a hashref of server properties (these may vary, you should C<Data::Dumper> to inspect). They will be provided by the RabbitMQ server to which you are connected.
+Get a reference to hash of server properties (these may vary, you should use C<Data::Dumper> to inspect). Properties will be provided for the RabbitMQ server to which you are connected.
```

Comment:
I tried re-writing the reference-using sentences and just couldn't do it.  Even the perlref tutorial does not use `hashref` as a noun, only a variable name as an apparent contraction for "hash reference".

As for linguistics, the editor in me says the places in the doc would be (were) wrong if `hashref` were swapped out for what I wrote because they all refer to what the hash contains.  What I mean is "Get a hashref of server properties" is wrong because the hashref does not have server properties, the hash it refers to does.  Likewise, "This method returns a hashref containing the following information:" is wrong because the hashref does not contain the information, the hash it refers to does.  They are just references, they do not _have_ or _contain_ anything, they _are_ a pointer/link/whatever to something else.

Thanks for your examples, I get the subtle difference and I get that hashref is in common usage.  I just think that hashref is a colloquial contraction for "hash reference", which is just a more concise way to write "reference to hash" and that there is not really anything magical or more informative about it.

If this is a blocker for merging I'll change it but I'll rewrite the sentences so that hashrefs are only references without stuff in them.